src/app/starredentries/starredentries.component.html

```diff
@@ -73,22 +76,52 @@
                 <ng-container
                   *ngTemplateOutlet="
                     tool.mode === 'DOCKSTORE_YML' ? workflowSummary : toolSummary;
-                    context: tool.mode === 'DOCKSTORE_YML' ? { workflow: tool, isAppTool: true } : { tool: tool }
+                    context: tool.mode === 'DOCKSTORE_YML'
+                      ? { workflow: tool, entryType: entryType.AppTool, isAppTool: true }
+                      : { tool: tool }
                   "
                 ></ng-container>
               </mat-card>
             </div>
           </div>
         </mat-tab>
 
-        <ng-template #workflowSummary let-workflow="workflow" let-isAppTool="isAppTool">
+        <mat-tab>
+          <ng-template mat-tab-label>
+            <div fxLayoutGap="0.5rem">
+              <img class="site-icons-tab" src="../../../assets/svg/sub-nav/notebook.svg" alt="notebook logo" />
+              <span>Notebooks</span>
+              <span class="tab-display" data-cy="starred-notebooks-count">{{ starredNotebooks?.length }}</span>
+            </div>
+          </ng-template>
+          <div>
+            <mat-card
+              class="alert alert-info mat-elevation-z starred-header"
+              data-cy="no-notebooks-banner"
+              role="alert"
+              *ngIf="starredNotebooks?.length === 0"
+            >
+              You have no starred notebooks.
+            </mat-card>
+            <div fxLayout="column" fxLayoutGap="1rem" class="mb-2 mx-1">
+              <mat-card fxFlex *ngFor="let notebook of starredNotebooks; let last = last" class="m-1 pb-3">
+                <ng-container
+                  *ngTemplateOutlet="workflowSummary; context: { workflow: notebook, entryType: entryType.Notebook, isAppTool: false }"
+                ></ng-container>
+              </mat-card>
+            </div>
+          </div>
+        </mat-tab>
+
+        <ng-template #workflowSummary let-workflow="workflow" let-entryType="entryType" let-isAppTool="isAppTool">
```

Comment:
Now that you're passing the `EntryType` (good!), you should be able to get rid of `isAppTool`. It's redundant and possibly contradictory (what if somebody passed EntryType.Notebook and isAppTool true)?

Reply:
I removed that before but I realized I needed it to add the conditional for https://github.com/dockstore/dockstore-ui2/blob/f34e653c63fd99e15c957561c584d3acb71d8577/src/app/starredentries/starredentries.component.html#L123

Reply:
Can you do `entryType === entryType.AppTool ? 'tool' : entryType`?

Reply:
No, it comes with an `Unresolved variable AppTool ` warning on intellij. Since the variable on line 116 is named to entryType, it looks like its checking `appTool === entryType.AppTool` and resulting to false. One possible fix is to rename the variable to something like `let-type="entryType"`. That way we can do `type === entryType.AppTool ? 'tool' : type`. Do you think that would be better than `isAppTool`?

Reply:
Yes, I think it's better than using `isAppTool` -- you already have all the info you need in the entryType parameter.

Renaming makes sense, I can see how the compiler would get confused between entryType defined in the .ts file vs entryType defined in the template. In fact, I wonder if it's actually using the entryType you think it's using on line 124 (maybe it is, maybe it's not). :)

Reply:
Changed